docs/release-notes/5.7.44-50.md

```diff
@@ -32,6 +33,18 @@ Percona Server for MySQL 5.7.44-50 contains the following fixes.
 
 * [PS-9132]: After an unexpected server exit, persistent information about executed Global Transaction Identifiers (GTIDs) was lost during the `Gtid_state::save` operation.
 
+## Third-party fixes
+
+The following Common Vulnerabilities and Exposures (CVE) records have been identified in third-party libraries:
+
+* [CVE-2023-6129]
+
+* [CVE-2024-0853]
```

Comment:
@patrickbirch Could you please add this info as well?
CVE-2023-6129 : 
OpenSSL versions 1.1.1 and 1.0.2 are not affected by this issue.
If OpenSSL 3.x.y is being used, upgrade to one of the 3.0.13, 3.1.5, 3.2.1 versions.
CVE-2024-0853 : 
8.5.0 is the only version affected. Rest of the versions(both higher and lower) are fine.